### PR TITLE
completed javascript-functions-pratice

### PR DESCRIPTION
**Pull request by BrianBozi**
image

### CODE REVIEW

File: javascript-functions-practice/get-character-at-index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/* exported getCharacterAtIndex */
+function getCharacterAtIndex(string, index) {
+  var CharacterAtIndex = string[index];
```
Comment (BrettAlbright) [inline javascript-functions-practice/get-character-at-index.js]: we do not want to use Capital letters for variable names except in special circumstances of which this is not one :)

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one tiny update!

File: javascript-functions-practice/get-character-at-index.js (hunk 1 above)
Comment (BrianBozi) [reply javascript-functions-practice/get-character-at-index.js]: ok got it!

**Comment by BrianBozi** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**